hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -610,12 +610,12 @@ def calc_is_empty(self, recur):
         # We assume that a bundle will grow over time
         return False
 
-    def _available(self, data):
+    def is_currently_empty(self, data):
```

Comment:
this is an inversion and a rename, which I think makes its purpose clearer via comparison to `is_empty`